docs/api-reference/blocks/create-block.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: "Create block"
+api: "POST /api/pipelines/:pipeline_uuid/blocks"
+---
+`POST /api/pipelines/:pipeline_uuid/blocks`
+
+Create a new block in one or more pipelines.
+
+<ParamField query="name" type="string" required>
+  Human readable block name. This is used to generate the UUID, e.g. `restless dust`. Names will be slugified to a safe-uuid.
+</ParamField>
+<ParamField query="type" type="string" required>
+  Type of block:
+  `callback`, `chart`, `custom`, `data_exporter`, `data_loader`, `extension`, `dbt`, `scratchpad`, `sensor`, `transformer`
```

Comment:
Still looks like this list is missing `conditional`, `markdown`, and `global_data_product`

Reply:
Whoops, forgot to add the change 🤦